README.md

```diff
@@ -0,0 +1,33 @@
+# DamagingBlocks
+
+## Goal: 
+To allow modders to create blocks which cause damage to players upon physically touching them or destroy objects which land on the blocks (i.e. Lava)
+
+## Usage:
+
+The DamagingBlocks module uses a component and a system. Essentially, a component is a request for an entity to have a feature, and a system is provides instructions for the entity to perform the feature.
+
+## DamagingBlockComponent
+
+Defines the rate and amount of damage which would be inflicted on the player.
+
+<pre>
+public class DamagingBlockComponent implements Component {
+    public int timeBetweenDamage = 1000; //The rate (value in milliseconds) at which the damage is inflicted
+    public int blockDamage = 20;   //The damage the block inflicts to the player
+    @Replicate(FieldReplicateType.SERVER_TO_OWNER)
+    public long nextDamageTime; //Helper variable for the system to know when to inflict damage
+}
+</pre>
+
+## DamageSystem
+
+The logic for inflicting damage to players to destroying blocks when the block is physically touched.
+
+### Methods in DamageSystem
```

Comment:
I would remove coverage of individual methods - that is much better to leave to javadoc comments. Getting the balance of documentation is tricky - too little and it won't be useful, too much and it'll go out of date when somebody updates the code yet forgets how detailed the documentation is and doesn't update that as well.

Instead you could see something like "For further details see javadoc in the https://github.com/Terasology/DamagingBlocks/blob/master/src/main/java/org/terasology/damagingblocks/DamageSystem.java " (you can make a nice link with `[text](link)`

... admittedly it seems like nobody wrote any good javadoc for that class, sigh :-) So you can actually take your comments for the two methods here and move them into DamageSystem as javadoc.